### PR TITLE
Chinese patch-2

### PR DESCRIPTION
**Pull request by kj415j45**
Add a changed section

### CODE REVIEW

**Comment by nanaya** [comment]
Hey can you update for all other languages as well?

**Comment by kj415j45** [comment]
use a new PR or just in this?

**Comment by nanaya** [comment]
just in this. Remove the ones that doesn't match.

**Comment by kj415j45** [comment]
:thinking: nekodex change all(except one) section in layout.menu.ranking(s)
image

~~I've to go to school now, will work on it tomorrow~~

**Comment by kj415j45** [comment]
oh, I know, you mean remove `layout.menu.ranking` for other translation?

**Comment by nanaya** [comment]
nevermind

File: resources/lang/es/layout.php [hunk 1]
```diff
@@ -54,14 +54,6 @@
             '_' => 'beatmapsets',
             'discussion' => 'modding',
         ],
-        'ranking' => [
```
Comment (nanaya) [inline resources/lang/es/layout.php]: should rename this to `rankings`, rename `getCharts` to `charts`, keep `_`, and remove everything else.

Comment (kj415j45) [inline resources/lang/es/layout.php]: I think that's not easy for their translator to find that here are some missing sections, remove section block may easy to find

Comment (nanaya) [inline resources/lang/es/layout.php]: at current system it doesn't make it easier

**Comment by nanaya** [comment]
Need styleci fix.

**Comment by kj415j45** [comment]
Also I did something wrong, fix it today

**Pull request merged**